models/issue_comment.go

```diff
@@ -870,3 +1060,72 @@ func DeleteComment(doer *User, comment *Comment) error {
 
 	return nil
 }
+
+func fetchCodeComments(e Engine, issue *Issue, currentUser *User) (map[string]map[int64][]*Comment, error) {
```

Comment:
might be more approachable to make this map a concrete type and explain it?

I guess its `file > line > coments`, right?